src/s3/util/fileViewerManager.ts

```diff
@@ -135,65 +138,46 @@ export class S3FileViewerManager {
         const targetLocation = vscode.Uri.file(targetPath)
 
         if (this.cacheArns.has(fileNode.file.arn)) {
-            //get it from temp IF it hasn't been recently modified, then return that
-
-            getLogger().debug(`cache is working!, found ${fileNode.file.key} in cache`)
-            showOutputMessage(`cache is working!, found ${fileNode.file.key} in cache`, this.outputChannel) //TODOD:: debug log remove
+            getLogger().info(`FileViewer: found file ${fileNode.file.key} in cache`)
 
-            //explorer (or at least the S3Node) needs to be refreshed to get the last modified date from S3
+            //Explorer (or at least the S3Node) needs to be refreshed to get the last modified date from S3
             const newNode = await this.refreshNode(fileNode)
             if (!newNode) {
-                showOutputMessage(
-                    `!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! something happeneded`,
-                    this.outputChannel
-                ) //TODOD:: debug log, remove
+                getLogger().error(`FileViewer: Error, refreshNode() returned undefined with file: ${fileNode.file.key}`)
+                getLogger().debug(`Cache contains: ${this.cacheArns}`)
                 return
             }
 
             fileNode = newNode
             const lastModifiedInS3 = fileNode!.file.lastModified
             const { birthtime } = fs.statSync(targetLocation.fsPath)
 
-            getLogger().debug(`last modified in S3: ${lastModifiedInS3}`)
-            showOutputMessage(`last modified in S3: ${lastModifiedInS3}`, this.outputChannel) //TODOD: debug log, remove
-            showOutputMessage(`creation date: ${birthtime}`, this.outputChannel)
-            getLogger().debug(`creation date: ${birthtime}`) //TODOD: debug log, remove
+            getLogger().debug(`FileViewer: File ${fileNode.file.name} was last modified in S3: ${lastModifiedInS3}`)
+            getLogger().debug(`FileViewer: Last cached download date: ${birthtime}`)
 
             if (lastModifiedInS3! <= birthtime) {
-                showOutputMessage(`good to retreive, last modified date is before creation`, this.outputChannel) //TODOD: debug log, remove
-                getLogger().debug(`good to retreive, last modified date is before creation`)
-
-                //await this.listTempFolder()
+                getLogger().info(`FileViewer: good to retrieve, last modified date is before creation`)
                 return targetLocation
             } else {
-                getLogger().debug(`last modified date is after creation date!!, removing file and redownloading`)
-                showOutputMessage(
-                    `last modified date is after creation date!!, removing file and redownloading`,
-                    this.outputChannel
-                ) //TODOD: debug log, remove
-
                 fs.unlinkSync(targetPath)
+                getLogger().warn(
```

Comment:
This isn't an invalid state that the toolkit is trying to reconcile, so I would consider this as an `info`-level log statement. This is an expected workflow, consider something like this if there's a clash, like writing to an S3 bucket if the file in S3 changed while editing a local copy.